docs/abi-spec.rst

```diff
@@ -345,3 +348,86 @@ would result in the JSON:
   "name":"foo",
   "outputs": []
   }]
+
+Use of Structs in Types
```

Comment:
And perhaps make a generic section about "tuple" encoding, where `name` is optional in case of tuples and is mandatory in case of structs (aka named tuples).

Reply:
The thing is that the type itself does not have named components, the names only appear in the JSON description of the interface. I can remove the term "tuple" from the documentation if that makes it easier to understand.